css/style.css

```diff
@@ -104,14 +104,27 @@ input {
 	width: 100%;
 	max-width: 500px;
 	border: 2px solid #000000;
-	margin: 10px 10px 10px 10px;
+	margin: 10px 0 10px 0;
```

Comment:
Using width 100% + margin on the left/right was causing the input boxes to be wider than the screen size creating horizontal scroll bars on mobile.